app/components/authentication_tokens/details/view.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= govuk_summary_card(title: name, html_attributes: { id: "token-#{id}" }) do |card|
+  card.with_action { govuk_link_to("Revoke", organisation_authentication_token_revoke_path(organisation_id: provider_id, authentication_token_id: id)) } if token.active?
```

Comment:
I think the link should be rendered only if the action to be performed is allowed by the policy.

Reply:
Would it not be redundant though? Right now you can only get to this page if you're already allowed to by the policy.

Reply:
Yes you're right not needed. Didn't notice where it is rendered 👍

Reply:
Might worth subclassing the existing policy though as if a token is already revoked it would allow revisiting the revoke page.